tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -45,19 +44,18 @@ void main(List<String> args) async {
 
 Future<void> _main(List<String> args) async {
   if (Platform.isWindows) {
-    print(
+    stdout.writeln(
```

Comment:
@vlidholt This still should be there? Aren't we supporting Windows full-fledged as Mac?